reDIP-SID/i2s_init_data.hex

```diff
@@ -55,8 +55,17 @@
 00 06 00 90
 
 // Input / Output routing
-00 0a 00 10 // Route I2S_IN to DAC
-00 24 00 27
+00 0a 00 70 // Route I2S_IN to DAP, route DAP to DAC
+
+// DAP EQ
+01 00 00 01 // Enable DAP
```

Comment:
I think the DAP will just be in passthrough mode with the GEQ stuff below commented out, but you might want to double check before merging

Reply:
If it doesn't work, we'll have to change the routing back.

Reply:
I'll merge first and can commit after if its not as expected. :+1: